config.toml

```diff
@@ -0,0 +1,8 @@
+main_backend = "main"
+
+[[backend]]
```

Comment:
Why the double square brackets `[[` and `]]`?

Reply:
Because it's a list backends, in haskell it creates `[Backend]`

Reply:
Gotcha, thanks!